package.json

```diff
@@ -69,10 +69,10 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@storybook/addon-actions": "^3.3.9",
-    "@storybook/addon-knobs": "^3.4.6",
+    "@storybook/addon-knobs": "^3.4.10",
     "@storybook/addon-links": "^3.3.9",
     "@storybook/react": "^3.3.9",
-    "autoprefixer": "^9.1.3",
+    "autoprefixer": "^9.1.5",
```

Comment:
Changelog: https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md

Seems like mostly under the good changes

Reply:
Plus the output of the base stylesheet compilation hasn't changed so (we'd see it in git) ... 👍 good enough for me